rpm/centos-7/docker-ce.spec

```diff
@@ -6,6 +6,7 @@ Group: Tools/Docker
 License: ASL 2.0
 Source0: engine.tgz
 Source1: cli.tgz
+Source2: metrics-plugin_%{_version}-stable-%{_plugin_version}.tgz
```

Comment:
can `-stable` be part of the `_plugin_version` so that an `edge` version of the plugin could be installed?